frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -854,6 +858,82 @@ export class HtmlDocument extends React.Component {
     this.setState({ translationTargetLanguageCode })
   }
 
+  handleClickAutoComplete = () => this.setState({
+    autoCompleteFormNewTagActive: false,
+    autoCompleteClicked: true
+  })
+
+  getMenuItemList = () => {
+    const { props, state } = this
+    const timelineObject = {
+      id: 'timeline',
+      label: props.t('Timeline'),
+      icon: 'fa-history',
+      children: state.config.apiUrl ? (
+        <Timeline
+          customClass={`${state.config.slug}__contentpage__timeline`}
+          customColor={state.config.hexcolor}
+          apiUrl={state.config.apiUrl}
+          loggedUser={state.loggedUser}
+          timelineData={state.timeline}
+          memberList={state.config.workspace.memberList}
+          newComment={state.newComment}
+          newCommentAsFileList={state.newCommentAsFileList}
+          disableComment={state.mode === APP_FEATURE_MODE.REVISION || state.mode === APP_FEATURE_MODE.EDIT || !state.content.is_editable}
+          availableStatusList={state.config.availableStatuses}
+          wysiwyg={state.timelineWysiwyg}
+          onChangeNewComment={this.handleChangeNewComment}
+          onRemoveCommentAsFile={this.handleRemoveCommentAsFile}
+          onValidateCommentFileToUpload={this.handleAddCommentAsFile}
+          onClickValidateNewCommentBtn={this.handleClickValidateNewCommentBtn}
+          onClickWysiwygBtn={this.handleToggleWysiwyg}
+          onClickRevisionBtn={this.handleClickShowRevision}
+          shouldScrollToBottom={state.mode !== APP_FEATURE_MODE.REVISION}
+          isLastTimelineItemCurrentToken={state.isLastTimelineItemCurrentToken}
+          key='Timeline'
+          invalidMentionList={state.invalidMentionList}
+          onClickCancelSave={this.handleCancelSave}
+          onClickSaveAnyway={this.handleClickValidateAnywayNewComment}
+          onInitWysiwyg={this.handleInitTimelineCommentWysiwyg}
+          showInvalidMentionPopup={state.showInvalidMentionPopupInComment}
+          searchForMentionInQuery={this.searchForMentionInQuery}
```

Comment:
I think it should be "mentionOrLink" here, maybe a merge problem

Reply:
Fixed in the merge I just did (it actually wasn't done)